python/ray/tests/test_chaos.py

```diff
@@ -123,6 +127,92 @@ def get(self):
     # assert_no_system_failure(p, 10)
 
 
+@pytest.fixture
+def set_kill_interval(request):
+    lineage_reconstruction_enabled, kill_interval = request.param
+    os.environ["RAY_lineage_pinning_enabled"] = ("1" if
+                                                 lineage_reconstruction_enabled
+                                                 else "0")
+    request.param = {
+        "kill_interval": kill_interval,
+        "head_resources": {
+            "CPU": 1
+        },
+        "worker_node_types": {
+            "cpu_node": {
+                "resources": {
+                    "CPU": 2,
+                },
+                "node_config": {},
+                "min_workers": 0,
+                "max_workers": 4,
+            },
+        },
+        # We just want the autoscaler to restart nodes, not take them down.
+        "idle_timeout_minutes": 10,
+    }
+    cluster = _ray_start_chaos_cluster(request)
+    while True:
+        try:
+            yield (lineage_reconstruction_enabled, kill_interval,
+                   next(cluster))
+        except StopIteration:
```

Comment:
When is it raised? Can you comment that here?

Reply:
Ah yeah, it's a bit awkward but I have to do this because the actual fixture is a generator.